server/norace_test.go

```diff
@@ -9793,6 +9801,13 @@ func TestNoRaceJetStreamSnapshotsWithSlowAckDontSlowConsumer(t *testing.T) {
 }
 
 func TestNoRaceJetStreamWQSkippedMsgsOnScaleUp(t *testing.T) {
+	streamCheckInterestStateInterval = 4 * time.Second
+	streamCheckInterestStateJitterInSeconds = 1
```

Comment:
The `mset.node.Propose(encodeStreamPurge(sp))` proposal could fail so need something to retry.
`mset.checkInterestState` is retrying, but has a too high interval to run in time during these tests. So lower them here for the test.